addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -100,8 +79,8 @@ These dynamic channels require the following parameters:
 |---------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Region Name   | String   | Region name. If the region is configured in the tracker app as well use the same name. Distance channels can also be defined as binding only regions (not configured in trackers) |
 | Region center | Location | Region center location                                                                                                                                                            |
-| Region Radius | Integer  | Geofence radius                                                                                                                                                                   |
-| Accuracy Threshold | Integer  | Location accuracy threshold (0 to disable)                                                                                                                                                                 |
+| Region Radius | Integer  | Geofence radius in meter                                                                                                                                                                 |
```

Comment:
This is not necessarily in meters. The binding uses Number:Length so it depends on system UOM.